### PR TITLE
Update Ruby SIG calendar links

### PR DESCRIPTION
**Pull request by fbogsany**
Point to the OpenTelemetry public calendar

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ You can also join us on [gitter](https://gitter.im/open-telemetry/opentelemetry-
...
+The Ruby SIG meets weekly on Wednesday alternating between [9 AM PDT](https://calendar.google.com/event?action=TEMPLATE&tmeid=MXVkNG9tb3NqZmdvMGpxOHZwNHE3MzB2NWVfMjAxOTA2MTlUMTYwMDAwWiBnb29nbGUuY29tX2I3OWUzZTkwajdiYnNhMm4ycDVhbjVsZjYwQGc&tmsrc=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com&scp=ALL) and [4 PM PDT](https://calendar.google.com/event?action=TEMPLATE&tmeid=MGw3azJocW9kbWRyNHZxMWVncjRrMWlsMTlfMjAxOTA2MjZUMjMwMDAwWiBnb29nbGUuY29tX2I3OWUzZTkwajdiYnNhMm4ycDVhbjVsZjYwQGc&tmsrc=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com&scp=ALL). This meeting is subject to change depending on contributors' availability, check the calendar ([web](https://calendar.google.com/calendar/embed?src=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com&ctz=America%2FLos_Angeles), [gCal](https://calendar.google.com/calendar?cid=Z29vZ2xlLmNvbV9iNzllM2U5MGo3YmJzYTJuMnA1YW41bGY2MEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t), [iCal](https://calendar.google.com/calendar/ical/google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com/public/basic.ics)) for specific dates.
```
Comment (SergeyKanzhelev) [inline README.md]: If we are just adding events to the public calendar - we can simply have a section on top talking about this calendar. May not be a part of this PR

Comment (SergeyKanzhelev) [inline README.md]: Can you use non-authenticated link for the calendar event? Like this: https://calendar.google.com/calendar/event?eid=MXVkNG9tb3NqZmdvMGpxOHZwNHE3MzB2NWVfMjAxOTA2MTlUMTYwMDAwWiBnb29nbGUuY29tX2I3OWUzZTkwajdiYnNhMm4ycDVhbjVsZjYwQGc&ctz=America/Los_Angeles

Just make sure in-private browsing will open it without the login form

Comment (fbogsany) [reply README.md]: 👍thank you!

Comment (fbogsany) [reply README.md]: I changed the calendar reference to point at the Community meeting section.

**Pull request merged**